src/pages/zoeken/[slug]/[ie]/index.tsx

```diff
@@ -425,9 +426,9 @@ const ObjectDetailPage: NextPage<ObjectDetailPageProps> = ({ title, url }) => {
 				.replaceAll('{title}', encodeURIComponent(mediaInfo?.name || ''))
 				.replaceAll('{title_serie}', encodeURIComponent(mediaInfo?.series?.[0] || ''));
 			window.open(resolvedFormUrl, '_blank');
-		} else {
-			setActiveBlade(MediaActions.RequestMaterial);
+			return;
 		}
+		setActiveBlade(MediaActions.RequestMaterial);
```

Comment:
Personal Preference: i like the original way of writing it more.
if the function has to return something anyway, i would accept your change, but explicitly returning, to then be able to remove the else doesn't really help readability imo.

Reply:
Okay, I'll put it back to the original format in my next PRs (since I'll be working in the same file).